client/my-sites/site-settings/disconnect-site/pagination-flow.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+class PaginationFlow extends Component {
+	verifySiteIsJetpack() {
+		if ( this.props.siteIsJetpack === false ) {
+			this.redirectToGeneral();
+		}
+	}
+
+	redirectToGeneral = () => {
```

Comment:
It seems like this is handled in the container component. Is it here just so that the `clickHandler` can do something?

Reply:
Yes, this is a redirect for 'Skip survey', which will fast forward the flow. The container component handles permissions. A comment that this is a placeholder would have been a good idea :)